src/web/screens/manageParticipants.tsx

```diff
@@ -20,19 +22,14 @@ import {
   UpdateParticipantForm,
 } from '../services/participant';
 import { GetAllParticipantTypes } from '../services/participantType';
-import { preloadSiteList } from '../services/site';
 import { RouteErrorBoundary } from '../utils/RouteErrorBoundary';
 import { PortalRoute } from './routeUtils';
 
+import '../components/ParticipantManagement/ParticipantManagement.scss';
```

Comment:
Other screens just have the `scss` file living along side the screen file, which makes sense to me since it is styling _around_ the components (i.e. that live in `components/ParticipantManagement`) rather than styling the components themselves. 

What do you thinkl?

Reply:
yeah, I agree.  I renamed and moved it.